http-tests/src/multi-jvm/scala/org/apache/pekko/http/PekkoHttpServerLatencyMultiNodeSpec.scala

```diff
@@ -145,13 +146,13 @@ class AkkaHttpServerLatencyMultiNodeSpec extends MultiNodeSpec(AkkaHttpServerLat
       })
   }
 
-  val enableSpec = system.settings.config.getBoolean("org.apache.pekko.test.AkkaHttpServerLatencySpec.enable")
+  val enableSpec = system.settings.config.getBoolean("pekko.test.PekkoHttpServerLatencySpec.enable")
```

Comment:
The problem was that these referenced the config above which had the keys still under `akka` (and also would never move to `org.apache.pekko`).